README.md

```diff
@@ -18,8 +18,18 @@ Add spree_ams to your Spree store's Gemfile:
 gem 'spree_ams', :github => 'hhff/spree_ams', :branch => '2-3-alpha'
 ```
 
+Make sure `config.action_controller.asset_host` is set in your environment config if not running in production.
```

Comment:
probably better to explicitly say `development.rb` here, as some people will probably be new to rails in general